provider/equinix/environ.go

```diff
@@ -732,12 +731,12 @@ func waitDeviceActive(ctx context.ProviderCallContext, c *packngo.Client, id str
 				return nil
 			}
 			if d.State == "failed" {
-				return errr.Wrap(ErrDeviceProvisioningFailed, fmt.Sprintf("device %s provisioning failed", id))
+				return fmt.Errorf("device %s provisioning failed: %w", id, ErrDeviceProvisioningFailed)
```

Comment:
Doesn't this change behaviour? `errors.Is` below will not be satisfied.

Reply:
No will work as advertised: https://go.dev/play/p/Pp899qYyUGq